### PR TITLE
community: Add save_model function for openvino reranker and embedding

### PR DESCRIPTION
**Pull request by openvino-dev-samples**

### CODE REVIEW

**Comment by OpenVINO-dev-contest** [comment]
Hi @baskaryan could you help to merge this PR, which removed some output messages in notebook document.

**Comment by OpenVINO-dev-contest** [comment]
@baskaryan @ccurme could you help to review and merge, thanks

**Comment by OpenVINO-dev-contest** [comment]
@efriis Could help to review, thanks

File: libs/community/langchain_community/embeddings/openvino.py [hunk 1]
```diff
@@ -295,14 +304,6 @@ class OpenVINOBgeEmbeddings(OpenVINOEmbeddings):
             )
     """
 
-    model_name_or_path: str
```
[the remark this answers is not in the document]
Comment (OpenVINO-dev-contest) [reply libs/community/langchain_community/embeddings/openvino.py:298]: Because we reuse these parameters from its super class.

**Review by ccurme — APPROVED** [review]

**Pull request merged**